spec/Conversions.tex

```diff
@@ -378,3 +378,27 @@ \subsection{Explicit Record Conversions}
 another record type \chpl{D} provided that \chpl{C} is derived
 from \chpl{D}.  There are no explicit record conversions that are not
 also implicit record conversions.
+
+
+\subsection{Explicit Type to String Conversions}
+\label{Explicit_Type_to_String_Conversions}
+\index{conversions!type to string}
+\index{conversions!explicit!type to string}
+
+A type expression can be explicitly converted to a \chpl{string}. The resultant
+\chpl{string} is the name of the type.
+
+\begin{chapelexample}{explicit-type-to-string.chpl}
+For example,
+\begin{chapel}
+var x: real(64) = 10.0;
+var s = x.type:string;
+\end{chapel}
+\begin{chapelpost}
+writeln(s);
+\end{chapelpost}
+\begin{chapeloutput}
+real(64)
+\end{chapeloutput}
+after running \chpl{s} will be the string \chpl{"real(64)"}.
```

Comment:
Only comment:  This doesn't quite parse for me:

    For example, <code> after running s will be the string "real(64)".  

How about something more like:  

    For example, after running the following code:  <code>  s will be the string "real(64)".  

Or, my preference (to avoid splitting a sentence across a code block):

    For example:  <code>  After running these statements, s will be the string "real(64)".

Final variation would be to have the program print out the result of the cast (or s) within the program and simply change the sentence following the code to:

    This program will print out the string "real(64)".

(Specifically, it feels a little funny to talk about what 's' will be after running the program ('s' won't exist after the program is run) and my dodge of talking about after running these statements feels a little squirrely...)